Utilities/General/test/BuildFile.xml

```diff
@@ -12,5 +12,5 @@
 <iftool name="cuda-gcc-support">
     <test name="CudaGCCSupport" command="echo Supported GCC version"/>
 <else/>
-    <test name="CudaGCCSupport" command="echo ERROR: Unsupported GCC version && exit 1"/>
+    <test name="CudaGCCSupport" command="echo ERROR: Unsupported GCC version &amp;&amp; exit 1"/>
```

Comment:
@gartung , please please `&&` with `;` 
This is a shell command and `echo ERROR: Unsupported GCC version &amp;&amp; exit 1` will fail